test/tests/fviks/diffusion/tests

```diff
@@ -8,4 +8,17 @@
     exodiff = 'test_out.e'
     ad_indexing_type = 'global'
   []
+  [diffusion-harmonic]
+    issues = '#21923'
+    design = 'FVDiffusionInterface.md'
+    requirement = 'The system shall be able to solve a diffusion problem with the domain split in half and two finite volume variables living on each side and interpolating the diffusivities using a harmonic mean.'
+    type = Exodiff
+    input = test.i
+    exodiff = 'harmonic.e'
```

Comment:
why is this file 49 kb ??? It's 1 D? ?

Reply:
No idea, I'll check.

Reply:
The other test file (with the average interpolation) is the same size. We have 2 variables in there. No idea why the exodus file is this large.

Reply:
In the NS module we have exodus gold files close to 1MB (or more).

Reply:
Resolving this as this is not really relevant to the content of this PR. Someone can create an issue if they want to dig into these sizes more